### ISSUE TITLE
[Color, typography and visual elements] Text doesn't meet AA compliance for color contrast. (06.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#73970]
#### VFS team name
Pension Benefits Team

#### VFS product name
Pension 527EZ Digital Form

#### VFS feature name
Form Reactivation

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Text doesn't meet AA compliance for color contrast.

### Issue Details
When hint text appears in the gray list-and-loop box (eg. on [Gross monthly income](https://staging.va.gov/pension/application/527EZ/financial/income-sources)), it doesn't meet color contrast requirements. I'm seeing a [contrast ratio of 4.02:1](https://webaim.org/resources/contrastchecker/?fcolor=71767A&bcolor=F0F0F0), and 4.5:1 would be the minimum here.

### VA.gov Experience Standard
[Category Number 06, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.4.3_AA
WCAG SC 1.4.6_AAA

### Platform Recommendation
I'm a little surprised we haven't noticed this in other products, and I have to think there are other places where this is an issue. If you haven't already, I'd recommend opening an issue with the Design System Team to address this globally.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by aplatt-coforma** [comment]
I can see the list loop item using the white background rather than the gray. 
<img width="773" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1hoQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8d4ee353bb14c18abd180327fa2d10da33fdb675/Screenshot%202024-01-19%20at%2010.44.40%E2%80%AFAM.png" alt="Screenshot 2024-01-19 at 10.44.40 AM.png" />
<img width="447" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN25oQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fe11cb57f3d57dfe3cc5fa370a4093a1adac5aad/Screenshot%202024-01-19%20at%2010.46.48%E2%80%AFAM.png" alt="Screenshot 2024-01-19 at 10.46.48 AM.png" />

**Comment by shiragoodman** [comment]
@aplatt-coforma are you asking for Governance to verify this fix before you close this ticket?

**Comment by aplatt-coforma** [comment]
@shiragoodman - no, we have verified the fix so good to go.
